modules/tinymce/src/core/main/ts/html/ParserFilters.ts

```diff
@@ -65,10 +64,6 @@ const createSafeEmbed = (mime?: string, src?: string, width?: string, height?: s
 const register = (parser: DomParser, settings: DomParserSettings): void => {
   const schema = parser.schema;
 
-  if (settings.remove_trailing_brs) {
```

Comment:
If we are just removing the `DomParserSetting` but keeping the editor option then this code needs to be moved somewhere else. Maybe move it to the `InitContentBody` since that is the only other spot where filters are added based on editor options. Then move the tests to something that acts on editors instead of the DOMParser.

Reply:
No wait it also registered in the `DomSerializerFilters` so we could just remove these lines and. move the test to the dom serializer.

Reply:
It seems that there are already tests for it it the `browser.tinymce.core.dom.SerializerTest` test but we need to make sure that it covers all the cases that the removed test does.

Reply:
I moved the tests from `modules/tinymce/src/core/main/ts/html/ParserFilters.ts` to `modules/tinymce/src/core/test/ts/browser/dom/SerializerTest.ts`